node/node.go

```diff
@@ -291,13 +291,26 @@ func (n *Node) GetKeyValuePairs(address string, options api.AccountQueryOptions,
 		return map[string]string{}, api.BlockInfo{}, nil
 	}
 
+	mapToReturn, err := n.getKeys(userAccount, ctx)
+	if err != nil {
+		return nil, api.BlockInfo{}, err
+	}
+
+	if common.IsContextDone(ctx) {
```

Comment:
Can this ever happen with `error == nil` from above when iterating? Or just in an edge case of micro seconds?